sass/_menu.scss

```diff
@@ -0,0 +1,74 @@
+.nav__header {
+    display: flex;
+    justify-content: space-between;
+
+    @media (min-width: 991px) {
+        max-width: 75rem;
+        margin: 0 auto;
+    }
+
+    .nav__logo {
```

Comment:
Pošto je __logo element nav bloka, podrazumjeva se da će biti korišten tako. Zato nema potrebe da se nav__logo nestuje unutar nav